docs/list-style-type.md

```diff
@@ -0,0 +1,57 @@
+> Typography
+
+# List Style Type
+
+Utilities for controlling the bullet/number style of a list.
+
+## Quick reference
+
+<qr-table />
+
+## Basic usage
+### Setting the list style type
+To create bulleted or numeric lists, use the `list-disc` and `list-decimal` utilities.
+
+<container>
+  <div class="flex flex-col gap-8 p-16">
+    <div>
+      <span class="s-text dark:s-text-inverted mb-3">list-disc</span>
+      <ul class="!list-disc !list-inside s-text dark:s-text-inverted !pl-0">
```

Comment:
It's a small thing, but just wondering - is `!` required in `!list-disc`?

Reply:
Not required, but it is needed in the example since the docs are styling lists and are winning otherwise.